DQM/SiPixelPhase1Summary/src/SiPixelPhase1Summary.cc

```diff
@@ -443,29 +460,31 @@ void SiPixelPhase1Summary::fillSummaries(DQMStore::IBooker& iBooker, DQMStore::I
           continue;  //Middle bins of y axis is empty
         if (i == (me_x->getNbinsY() / 2) + 1)
           continue;
+        if (me_x->getBinEntries(j + (me_x->getNbinsX() + 2) * i) < 30)  //Fill only if number of hits is above 30
+          continue;
```

Comment:
Is this cut expected to not create the residual histograms in the case of reduced occupancy/low acceptance?
I am asking since last test still shows histograms with null residual for Cosmic wf 7.3
https://tinyurl.com/sm5fsuu

Reply:
@dmeuser, in the interest of readability, can this "30" be a named constant rather than a magic number?

Reply:
@schneiml @jfernan2, may I ask if there is a method of `MonitorElement` to get the bin index of a 2D histogram to avoid this kludgy `binx+(TOTbinX+2)*biny`, something like https://root.cern.ch/doc/master/classTH2.html#a9d47a53d1b2499475542b70a183c5c1c?

Reply:
@mmusich @dmeuser It seems we have `getBinContent` and `getBinError` in 2D, but not `getBinEntries`. Feel free to add an overload here: https://github.com/cms-sw/cmssw/blob/master/DQMServices/Core/src/MonitorElement.cc#L643

Reply:
@schneiml I am currently trying to add the method to the MonitorElement. Compiling the code does work fine, but when running the Harvesting I always get something like this:

Reply:
There is clearly some level of memory corruption there. Not much else that I can tell without the code.

Reply:
I did not take the dependencies into account, my fault.